app/views/planning_applications/_validation_notice.html.erb

```diff
@@ -27,10 +27,9 @@
         We may request additional information and/or revisions before deciding whether the application should be recommended for permission or refusal.
       </p>
       <p class="govuk-body">
-        We aim to issue a decision by <%= @planning_application.expiry_date %>.
-        However, if your application has not been determined by <%= @planning_application.expiry_date %>,
-        you have the right to appeal to the Secretary of State, either online at https://www.gov.uk/government/organisations/planning-inspectorate,
-        or by post to Temple Quay House, 2 The Square, Temple Quay, Bristol BS1 6PN.
+        We aim to issue a decision by <%= @planning_application.expiry_date.to_date.to_fs %>.
+        However, if your application has not been determined by <%= @planning_application.expiry_date.to_date.to_fs %>,
+        you have the right to appeal to the Secretary of State, either online at <a href="https://www.gov.uk/government/organisations/planning-inspectorate" class="govuk-link" target="_blank">https://www.gov.uk/government/organisations/planning-inspectorate</a>, or by post to Temple Quay House, 2 The Square, Temple Quay, Bristol BS1 6PN.
```

Comment:
How about using the helper? Mainly because I think it adds a couple of extra attributes for a new tab, but also a little less repetitive.

```suggestion
        you have the right to appeal to the Secretary of State, either online at <%= govuk_link_to "https://www.gov.uk/government/organisations/planning-inspectorate", new_tab: true %>, or by post to Temple Quay House, 2 The Square, Temple Quay, Bristol BS1 6PN.
```